pio/st7789_lcd/st7789_lcd.c

```diff
@@ -28,6 +28,8 @@
 #define PIN_RESET 4
 #define PIN_BL 5
 
+#define SPI_POLARITY 0 // default to 0, some displays require polarity 1
```

Comment:
Minor nitpick, but perhaps this should be `SPI_CLOCK_POLARITY` to make it more obvious what it's actually changing?

Reply:
Changed.